samples/app-anonymous-users/nodejs/ClientApp/src/router/router.js

```diff
@@ -0,0 +1,38 @@
+// <copyright file="router.js" company="Microsoft Corporation">
```

Comment:
Typically this would just be named App.js.

Reply:
Fixed